.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run Link Checker
+on: [push]
+
+jobs:
+  initialize:
+    name: Clone Theme
+    runs-on: ubuntu-latest
+    steps:
+    - name: Initialize Theme
+      run: |
+        sudo apt-get update && sudo apt-get install -y git
+  build:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      env:
+        HUGO_VERSION: 0.74.3
```

Comment:
Can we look this up from the Netlify configuration?

(I think that's feasible).

Reply:
I'm updating it in the next commit.